CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ FetchContent_Declare(llhttp
     URL "https://github.com/nodejs/llhttp/archive/refs/tags/release/v8.1.0.tar.gz")
 FetchContent_Declare(UrlLib
     GIT_REPOSITORY https://github.com/BabylonJS/UrlLib.git
-    GIT_TAG c7c162be129e170a12bb8766c727b59c42853826)
+    GIT_TAG 585987f609318d328126ea20c298c9368bdfd91d)
```

Comment:
I guess UrlLib is technically slight out of date with arcana.cpp now.

Reply:
With CMake, the last FetchContent_Declare wins, but we should update the dependencies to be consistent. I'll make some updates.